### ISSUE TITLE
Markbot Error

### ISSUE THREAD

**Issue by Hath0013** [Hath0013/confectionery-website#11]
Hey Thomas, I'm having this error with my code. I think it's got something to do with the fact that I have two separate navigation menus. I remember having a similar issue last year.
I tried giving the header nav a, a width and it didn't do anything. Previously I had just a background colour for the hover state and no width set. Melissa and I couldn't figure it out so she told me to ask you.

@thomasjbradley

![screen shot 2016-12-14 at 8 31 36 pm](https://cloud.githubusercontent.com/assets/16677645/21244518/036ed832-c2ec-11e6-8165-5f59faf7b0d8.png)

**Comment by thomasjbradley** [comment]
Yeah, you're right, it's because you have two completely separate navigations.
Markbot is latching onto the first nav but it’s hidden on large screens.
